packages/appstore/src/constants/tour-steps-config-new.tsx

```diff
@@ -1,9 +1,9 @@
-import { Step, Styles, Locale } from 'react-joyride';
```

Comment:
Why is it `tour-steps-config-new` instead of `tour-steps-config`?

Reply:
`tour-steps-config-new `is a revised version of `tour-steps-config`. I deleted `tour-step-config` because it is not needed anymore because it was replaced by `tour-steps-config-new`